### PR TITLE
Update popper.js

### PR DESCRIPTION
**Pull request by ishanray**

### CODE REVIEW

File: src/popper.js [hunk 1]
```diff
@@ -142,10 +142,10 @@
 
         // if the popper variable is a configuration object, parse it to generate an HTMLElement
         // generate a default popper if is not defined
-        var isNotDefined = popper === undefined || popper === null;
+        var isNotDefined = typeof popper === undefined || popper === null;
```
Comment (FezVrasta) [inline src/popper.js]: typeof will return a string if I'm not wrong.

Comment (smashercosmo) [inline src/popper.js]: Why not just ```popper == null```?

Comment (FezVrasta) [inline src/popper.js]: I don't like such kind of generic comparisons

Comment (ishanray) [inline src/popper.js]: `typeof popper === 'undefined'`

In case popper is not defined then that check will throw an error.

**Comment by FezVrasta** [comment]
May you add a test to cover the code you changed?

**Comment by FezVrasta** [comment]
Thank you, awesome!

If you may be so kind to fix the conflicts I can then proceed to merge this PR into master.

**Comment by FezVrasta** [comment]
Great thanks.

**Pull request merged**